FirebaseTestProject/Views/PostsList.swift

```diff
@@ -9,10 +9,15 @@ import SwiftUI
 
 struct PostsList: View {
     @StateObject var postData = PostData()
+    @State private var searchText: String = ""
     
     var body: some View {
+        //Searchable Posts
+        SearchBar(text: $searchText)
```

Comment:
If the `SearchBar` is included this way, it appears as a separate tab from the rest of the `PostsList`. This is probably because SwiftUI is reading the `SearchBar` and the `NavigationView` as two separate tabs.

<img src="https://user-images.githubusercontent.com/34844819/130536051-e3013f52-293c-4a14-9632-5299a589cff1.png" width="200px" />
<img src="https://user-images.githubusercontent.com/34844819/130536058-a0d1b22d-9ac3-4393-93bf-8b5934f66b1d.png" width="200px" />

One way to resolve this behavior would be to move the  `SearchBar` into the `List` element below, and display the posts using a `ForEach` loop instead. I’m open to other solutions, though.